config/samples/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 ## Append samples you want in your CSV to this file as resources ##
 resources:
 - sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml
-- sw.kogito_v1alpha08_kogitoserverlessbuild.yaml
```

Comment:
Aren't needed all three files ?

Reply:
No they are not mandatory files.
Since that example is blank and the Build CR is created behind the scene from the Operator, at the moment that file is not very useful.

Reply:
ok